src/public/app/app.module.ts

```diff
@@ -7,6 +7,8 @@ import { AppComponent } from './app.component';
 import { CoreModule } from './core/core.module';
 import { SharedModule } from './shared/shared.module';
 import { DashboardComponent } from './features/dashboard/dashboard.component';
+import { UsersComponent } from './features/users/users.component';
```

Comment:
These two don't need to be imported here. They're behind a lazy loaded route and importing them here will cause them to be added to the bundle sent to the browser on every page load.